packages/instrumentation-express/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export { ExpressInstrumentation } from './express';
+export { ExpressInstrumentationConfig, ExpressRequestHookInformation } from './types';
```

Comment:
maybe export `ExpressInstrumentationAttributes` and `RequestHook` as well?

Reply:
good point. added
Thanks